### PR TITLE
add get_system_status

### PR DESCRIPTION
**Pull request by asmodehn**

### CODE REVIEW

File: lib/binance.ex [hunk 1]
```diff
@@ -221,6 +221,18 @@ defmodule Binance do
     end
   end
 
+  @doc """
+  Fetches system status.
+
+  Returns `{:ok, %Binance.SystemStatus{}}` or `{:error, reason}`.
+  """
+  def get_system_status() do
+    case HTTPClient.get_binance("/wapi/v3/systemStatus.html") do
```
Comment (Cinderella-Man) [inline lib/binance.ex:230]: As per my comment `wapi` endpoints are getting delisted but we could use `/sapi/v1/system/status`

File: test/binance_test.exs [hunk 2]
```diff
@@ -80,6 +80,13 @@ defmodule BinanceTest do
     end
   end
 
+  test "get_system_status success returns the system status" do
+    use_cassette "get_system_status_ok" do
+      assert {:ok, response} = Binance.get_system_status()
+      assert %Binance.SystemStatus{} = response
```
Comment (Cinderella-Man) [inline test/binance_test.exs:86]: Should data inside the struct be checked here?

**Review by Cinderella-Man — CHANGES_REQUESTED** [review]
Hello :wave: 

Thank you very much for your PR. I checked the endpoint and it looks like `wapi` endpoints are getting delisted:

https://www.binance.com/en/support/announcement/f45dde7da58b473aa885349946bed269

You could just update the URL in the PR to `/sapi/v1/system/status `(https://binance-docs.github.io/apidocs/spot/en/#wallet-endpoints) and it should be fine.

I added 2 comments inline.

**Comment by asmodehn** [comment]
Good catch, I hadn't noticed, thanks !

**Comment by Cinderella-Man** [comment]
> Good catch, I hadn't noticed, thanks ! (deprecation date was indeed right there in the cassette data)

Ha! I missed that `Deprecation` field in the response - good catch as well! This looks good and I believe it can be merged now.

Thank you very much once again for your contribution :+1:

**Pull request merged**